app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(players: list):
+    return sum(player.get_rating() for player in players)
+
+
+def elves_concert(elves: list):
+    for elf in elves:
```

Comment:
Here and on $11,  you could use list comprehension, but it's up to you.
*[elf.play_elf_song() for elf in elves]